modules/sonobiBidAdapter.js

```diff
@@ -78,6 +78,12 @@ export const spec = {
       payload.hfa = `PRE-${validBidRequests[0].crumbs.pubcid}`;
     }
 
+    if (deepAccess(validBidRequests[0], 'userId.tdid')) {
+      payload.tdid = validBidRequests[0].userId.tdid;
+    } else if (deepAccess(validBidRequests[0], 'crumbs.tdid')) {
```

Comment:
To clarify - is this backup meant to grab the information from the `pubCommonId` module?

If so, the property should be `crumbs.pubcid` instead of `crumbs.tdid`.

Reply:
No it is not meant to try to grab pubcid. I wasnt sure if tdid would be available on `crumbs`, so I handled that scenario just incase.

Reply:
@jsnellbaker  any update?

Reply:
@JonGoSonobi Sorry about delayed response here; the `crumbs` object won't be created by the `userID` module - it's only set by the `pubCommonId`.

Since the code would never find this object, I would suggest to remove this back-up if statement just to make your code clear.

Reply:
@jsnellbaker sounds good. done.